.github/workflows/log-viewer-docs.yaml

```
@@ -2,6 +2,8 @@ name: "docs"
 
 on:
   pull_request:
+    paths:
+        - "docs/**"
```

it seems only files should matter, and adding the "*" seems to be our convention
```suggestion
        - "docs/**/*"
```